src/test/scala/uk/gov/nationalarchives/dp/client/ClientTest.scala

```diff
@@ -0,0 +1,460 @@
+        <Entity title="page1File.txt" ref="8a8b1582-aa5f-4eb0-9c5d-2c16049fcb91" type="IO">http://localhost/page1/object</Entity>
+      </Entities>
+      <Paging>
+        <Next>http://localhost:{
+      port
+    }/api/entity/entities/updated-since?date=2023-04-25T00%3A00%3A00.000Z&amp;start=100&amp;max=100</Next>
+      </Paging>
+    </EntitiesResponse>
+    val secondPage = <EntitiesResponse>
+      <Entities>
+        <Entity title="page2File.txt" ref="6ca62825-4225-4dad-ac93-1d018bade02f" type="SO" deleted="true">http://localhost/page2/object</Entity>
+      </Entities>
+      <Paging>
+      </Paging>
+    </EntitiesResponse>
+
+    preservicaServer.stubFor(post(urlEqualTo(tokenUrl)).willReturn(ok(tokenResponse)))
+    preservicaServer.stubFor(
+      get(urlPathMatching(s"/api/entity/entities/updated-since"))
+        .withQueryParams(
+          Map(
+            "date" -> equalTo("2023-04-25T00:00:00.000Z"),
+            "max" -> equalTo("100"),
+            "start" -> equalTo("0")
+          ).asJava
+        )
+        .willReturn(ok(firstPage.toString()))
+    )
+    preservicaServer.stubFor(
+      get(urlPathMatching(s"/api/entity/entities/updated-since"))
+        .withQueryParams(
+          Map(
+            "date" -> equalTo("2023-04-25T00:00:00.000Z"),
+            "max" -> equalTo("100"),
+            "start" -> equalTo("100")
+          ).asJava
+        )
+        .willReturn(ok(secondPage.toString()))
+    )
+    val client = testClient(s"http://localhost:$port")
+    val response = valueFromF(client.entitiesUpdatedSince(date, authDetails))
+
+    val pageOne = response.head
+    val pageTwo = response.last
+
+    pageOne.ref.toString should equal("8a8b1582-aa5f-4eb0-9c5d-2c16049fcb91")
+    pageOne.path should equal("information-objects")
+    pageOne.title should be("page1File.txt")
+    pageOne.deleted should be(false)
+
+    pageTwo.ref.toString should equal("6ca62825-4225-4dad-ac93-1d018bade02f")
+    pageTwo.path should equal("structural-objects")
+    pageTwo.title should be("page2File.txt")
+    pageTwo.deleted should be(true)
+  }
+
+  "entitiesUpdatedSince" should "return an empty list if none have been updated" in {
+    val date = ZonedDateTime.of(2023, 4, 25, 0, 0, 0, 0, ZoneId.of("UTC"))
+    val input = <EntitiesResponse>
+      <Entities></Entities>
+    </EntitiesResponse>
+
+    preservicaServer.stubFor(post(urlEqualTo(tokenUrl)).willReturn(ok(tokenResponse)))
+    preservicaServer.stubFor(
+      get(urlPathMatching(s"/api/entity/entities/updated-since"))
+        .withQueryParams(
+          Map(
+            "date" -> equalTo("2023-04-25T00:00:00.000Z"),
+            "max" -> equalTo("100"),
+            "start" -> equalTo("0")
+          ).asJava
+        )
+        .willReturn(ok(input.toString()))
+    )
+
+    val client = testClient(s"http://localhost:$port")
+    val response = valueFromF(client.entitiesUpdatedSince(date, authDetails))
+
+    response.size should equal(0)
+  }
+}
```

Comment:
Should we add a test for the 400 error returned from this endpoint to see what happens if you enter in a malformed date, negative number or too large of a number for the 'start' and 'max'?